docs/dev-tools/chopsticks/index.md

```diff
@@ -0,0 +1,395 @@
+---
+title: Using chopsticks for Substrate based chains.
+description: Configure and fork Substrate-based blockchains locally with Chopsticks. Learn installation, configuration, and usage for streamlined blockchain development.
+---
+
+# Chopsticks
+
+## Introduction
+
+[Chopsticks](https://github.com/AcalaNetwork/chopsticks/){target=_blank}, created and maintained by the [Acala Foundation](https://github.com/AcalaNetwork), is a powerful tool designed to enhance the development process for Substrate-based blockchains. It offers developers a user-friendly method to locally fork existing chains, enabling them to:
+
+- Experiment with custom blockchain configurations in a local environment
+- Replay blocks and analyze how extrinsics affect state
+- Fork multiple blocks for comprehensive XCM testing
+
+With Chopsticks, developers can simulate and test complex blockchain scenarios without deploying to a live network. This tool significantly reduces the complexity of building blockchain applications on Substrate, making it more accessible to developers of varying experience levels. Ultimately, Chopsticks aims to accelerate innovation in the Substrate ecosystem by providing a robust, flexible testing framework.
+
+!!!note
+    Chopsticks currently does not support Ethereum JSON-RPC calls, so you cannot use it to fork your chain and connect Metamask.
+
+## Prerequisites
+
+Before you begin, ensure you have the following installed:
```

Comment:
Just nit-picking; maybe you can add here WHY it's necessary to have these elements installed